docs/docs/tags.mdx

```diff
@@ -1760,6 +1760,24 @@ Additionally, this tag does nothing when in VR.
   </PossibleValueCode>
 </PossibleValuesTable>
 
+### `defaultLighting`
+<Badges>
+    <MapPortalBadge/>
```

Comment:
@TroyceGowdy These should be the `GridPortalBadge` and the `MiniGridPortalBadge`